presto-hive/src/main/java/com/facebook/presto/hive/HiveCommitHandle.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.spi.connector.ConnectorCommitHandle;
+
+// TODO: Add more concrete details, e.g., input and output data commit times.
+public class HiveCommitHandle
```

Comment:
We don't need this file right? Or, we should rename `HiveMetastoreCommitHandle` into `HiveCommitHandle` and remove this class from presto-hive

Reply:
Yeah, for the completeness of this diff, I put it here. In the end, this should be moved into presto-hive-metastore, with name HiveMetastoreCommitHandle. I can remove it now.

Reply:
Let's remove this class and rename `HiveMetastoreCommitHandle` to `HiveCommitHandle `